packages/cozy-harvest-lib/src/components/EditAccountModal.jsx

```diff
@@ -23,7 +23,7 @@ import { withMountPointHistory } from './MountPointContext'
 import logger from '../logger'
 import { withTracker } from './hoc/tracking'
 import useTimeout from './hooks/useTimeout'
-import { withRouter } from 'react-router'
+import { withRouter } from 'react-router-dom'
```

Comment:
This one, why not. https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/package.json#L87 

But I think it'll crash on cozy-drive because it imports harvest without respecting the peerDep...